<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbDialog</class>
 <widget class="QDialog" name="DbDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection Manager</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Clic on a connection to edit it</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Connection settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="hostLabel">
        <property name="text">
         <string>Host</string>
        </property>
        <property name="buddy">
         <cstring>hostEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="hostEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="userLabel">
        <property name="text">
         <string>Username</string>
        </property>
        <property name="buddy">
         <cstring>userEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="userEdit"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="saveCheckBox">
        <property name="text">
         <string>Save password</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="pswdLabel">
        <property name="text">
         <string>Password</string>
        </property>
        <property name="buddy">
         <cstring>passEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="passEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="dbLabel">
        <property name="text">
         <string>Database</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="dbEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="dbBrowseButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Database type</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="DbComboBox" name="dbTypeComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="testButton">
        <property name="text">
         <string>Test connection</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="resultLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;- Clic to test</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Alias</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="aliasEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="dbListView"/>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/img/edit_add.png</normaloff>:/img/edit_add.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="remButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="toggleButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DbComboBox</class>
   <extends>QComboBox</extends>
   <header>../src/widgets/dbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>hostEdit</tabstop>
  <tabstop>userEdit</tabstop>
  <tabstop>saveCheckBox</tabstop>
  <tabstop>passEdit</tabstop>
  <tabstop>dbEdit</tabstop>
  <tabstop>dbTypeComboBox</tabstop>
  <tabstop>testButton</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>dbListView</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>remButton</tabstop>
  <tabstop>toggleButton</tabstop>
  <tabstop>dbBrowseButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>saveCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>pswdLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>passEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DbDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DbDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
